### ISSUE TITLE
DefaultWindowManager not Full path as stated

### ISSUE THREAD

**Issue by atorregr** [neutrinolabs/xrdp#1143]
Hello.

I am using xrdp-0.9.6-4.el7.x86_64 and xorgxrdp-0.2.6-2.el7.x86_64 on top of RHEL7.4. In sesman.ini's man page the following is stated:
       DefaultWindowManager=filename
              Full path to the default startup script used by xrdp-sesman to start a session if the user script
              is disabled or missing.

However, this does not appear to be so, in the sense that the path does not appear to be full but relative to /usr/libexec/xrdp; upon configuring sesman.ini to include
DefaultWindowManager=/opt/oss/bin/tier1_xrdp.sh
I was receiving these errors in sesman's log:
[20180606-17:36:29] [DEBUG] errno: 2, description: No such file or directory
[20180606-17:36:29] [DEBUG] execlp3 parameter list:
[20180606-17:36:29] [DEBUG]         argv[0] = /usr/libexec/xrdp//opt/oss/bin/tier1_xrdp.sh
[20180606-17:36:29] [DEBUG]         argv[1] = /opt/oss/bin/tier1_xrdp.sh
[20180606-17:43:22] [INFO ] shutting down sesman 1
I tried to workaround the issue by stepping up in the directory tree, but then I found another error, maybe caused by a limitation in the variable's size; configuring sesman to include
DefaultWindowManager=../../../opt/oss/bin/tier1_xrdp.sh
I was receiving these errors in sesman's log:
[20180606-17:45:04] [DEBUG] errno: 2, description: No such file or directory
[20180606-17:45:04] [CORE ] waiting for window manager (pid 6363) to exit
[20180606-17:45:05] [DEBUG] execlp3 parameter list:
[20180606-17:45:05] [DEBUG]         argv[0] = /usr/libexec/xrdp/../../../opt/oss/bin/tier1_xrdp
[20180606-17:45:05] [DEBUG]         argv[1] = ../../../opt/oss/bin/tier1_xrdp

I finally worked around the issue creating a link /usr/libexec/xrdp/tier1_xrdp.sh -> /opt/oss/bin/tier1_xrdp.sh and configuring DefaultWindowManager=tier1_xrdp.sh. However, I don't believe creating a link (or any file) under a product's directory is the best solution. One of the two previous approaches should work, preferrably being consistent with the man page. Or maybe it could be either relative or absolute depending on whether it starts with a "/" or not.

Best regards.

**Comment by metalefty** [comment]
Thanks for reporting! I'll make an effort to fix it until the next release v0.9.7.

**Comment by atorregr** [comment]
You're very welcome.

BR

**Comment by metalefty** [comment]
Also, man page of sesman.ini wasn't correct.
```
       DefaultWindowManager=filename
              Full path to the default startup script used by xrdp-sesman to
              start a session if the user script is disabled or missing.
```

**Comment by metalefty** [comment]
@atorregr Can you try & test #1147?

**Comment by metalefty** [comment]
Will be included in v0.9.7, test it then.

**Comment by atorregr** [comment]
Will do, thanks a lot.

BR

**Comment by metalefty** [comment]
Solved in v0.9.7.
